### PR TITLE
Add govcloud support

### PR DESCRIPTION
**Pull request by enxebre**
This introduces support for aws govcloud:
- Duplication has been generally chosen over abstraction to keep it as simple as possible and being non intrusive with conventional aws platform for now. 
- As a follow up, both internal aws and govcloud could use the same bastion infrastructure created here contrib/govcloud
- Jenkins credentials need to be setup

### CODE REVIEW

**Comment by cpanato** [comment]
ok to test

File: modules/dns/powerdns/variables.tf [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  description = "List of string IPs for k8s API"
+  type        = "list"
+}
+
+variable "self_hosted_etcd" {
+  default     = ""
+  description = "See tectonic_self_hosted_etcd in config.tf"
+}
+
+variable "tectonic_vanilla_k8s" {
```
Comment (sym3tri) [inline modules/dns/powerdns/variables.tf]: Will need to remove these once your other PR lands.

**Comment by cpanato** [comment]
retest this please

**Comment by cpanato** [comment]
retest this please

**Comment by enxebre** [comment]
retest this please

**Comment by enxebre** [comment]
retest this please

**Comment by enxebre** [comment]
retest this please

**Comment by cpanato** [comment]
waiting for @sym3tri / @alexsomesan

**Comment by sym3tri** [comment]
ship it

**Comment by sym3tri** [comment]
@enxebre don't forget to cherry-pick into track-1

**Comment by enxebre** [comment]
track-1 PR https://github.com/coreos/tectonic-installer/pull/2660

**Pull request merged**